### PR TITLE
Removed a link for the Register Button

### PR DESCRIPTION
**Pull request by akashyerra**
Please review this PR and commit the changes made

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,7 +48,7 @@
             <li><a href="#footer" class="home">CONTACT US</a></li>
             <!-- <div class="about-cursor"></div> -->
             <li>
-                <a href="https://unstop.com/o/qTPjOK9?lb=2ZvsJH5"  class= "active headRegister" target="_blank">REGISTER</a>
+                <a href="" class= "active headRegister" >REGISTER</a>
```
Comment (kvenkatamar) [inline index.html:51]: @akashyerra can you change the color of the button also??

**Review by kvenkatamar — CHANGES_REQUESTED** [review]
@akashyerra Can you change the color of the button to grey?

File: index.html (hunk 1 above)
Comment (akashyerra) [reply index.html:51]: Yeah! I have already changed the color of the register button to grey please check my another PR once

**Review by kvenkatamar — APPROVED** [review]
I am approving the changes

**Pull request merged**